content/highlight.css

```diff
@@ -0,0 +1 @@
+<%= Pygments.css(style: "tango") %>
```

Comment:
would make more sense to me to add pygments style overrides here below the Pygments?